hub/constants.py

```diff
@@ -1,4 +1,17 @@
 BYTE_PADDING = b"\0"
+
+# number of bytes per unit
+KB = 1000
+MB = 1000 * KB
+GB = 1000 * MB
+
+MIN_MEMORY_CACHE_SIZE = 32 * MB
+MIN_LOCAL_CACHE_SIZE = 160 * MB
+
 CHUNKS_FOLDER = "chunks"
 META_FILENAME = "meta.json"
 INDEX_MAP_FILENAME = "index_map.json"
+
+PYTEST_MEMORY_PROVIDER_BASE_ROOT = "hub_pytest"
+PYTEST_LOCAL_PROVIDER_BASE_ROOT = "/tmp/hub_pytest/"
```

Comment:
I foresee an `issue: tests do not run on my machine` in your future... :crystal_ball:

Reply:
i also was thinking of this for windows

Reply:
let's cross that bridge when we get there

Reply:
since we dont have windows running yet anyways, let's address that in a new PR

Reply:
lets at least add a todo for this